scripts/create_pelorus_operator

```diff
@@ -42,17 +39,19 @@ function print_help() {
     printf "\t  -s\tpath to source pelorus helm charts folder, default: %s\n" "${DEFAULT_PELORUS_CHARTS_DIR}"
     printf "\t  -d\tpath to EMPTY destination folder, default: %s\n" "${DEFAULT_PELORUS_OPERATOR_DIR}"
     printf "\t  -p\tpath to DIR with operator patches to be applied, default: %s\n" "${DEFAULT_PELORUS_OPERATOR_PATCHES_DIR}"
+    printf "\t  -v\toperator version to be used in a format x.y.z. If not set z+1 is used.\n"
```

Comment:
instead of x, y and z, would be better to use major, minor and patch?

Reply:
I think it's better to stick with x.y.z because SemVer allows to have some other parts then just a non negative number and we really do want to stick with numbers only.

https://semver.org/#spec-item-2

Reply:
Instead of passing the version we want (that can lead to versioning errors), should we accept a list of 3 options (major, minor and patch) and apply versioning rules according to the option? (and I would let patch as default like already is)

Reply:
If @mateusoliveira43 is satisfied then this looks mergeable!